src/main/java/c3po/command/CommandEnum.java

```diff
@@ -0,0 +1,41 @@
+package c3po.command;
+
+/**
+ * Represents the command that the user inputs.
+ */
+public enum CommandEnum {
+    GREET, BYE, LIST, MARK, UNMARK, TODO, DEADLINE, EVENT, DELETE, FIND, UNKNOWN, INVALID;
+
+    /**
+     * Returns the Command enum value corresponding to the specified string.
+     *
+     * @param command The string representation of the command.
+     * @return The Command enum value corresponding to the specified string.
+     */
+    public static CommandEnum fromString(String command) {
+        try {
+            return CommandEnum.valueOf(command.toUpperCase());
+        } catch (IllegalArgumentException e) {
+            return UNKNOWN;
```

Comment:
Having UNKNOWN be a valid enum value is possibly error-prone since it's not very type-safe. Perhaps you could return an Optional<CommandEnum> instead?